docs/docs/features/studio.md

```diff
@@ -10,6 +10,37 @@ ZMK Studio is in beta. Although every effort has been made to provide a stable e
 
 ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards. Studio is still under active development, and is not yet ready for casual end user use.
 
+## Capabilities
+
+ZMK Studio currently has the following capabilities:
+
+| Legend: | ✅ Supported | 🚧 Under Development | 💡 Planned | 🕯️ Low Priority | ❌ Not Planned |
+| :------ | :----------- | :------------------- | :--------- | :-------------- | -------------- |
+
+| Studio Status | **Feature or Capability**                                                                         |
+| :-----------: | ------------------------------------------------------------------------------------------------- |
+|      ✅       | Making keymap changes while keyboard is in use                                                    |
+|      ✅       | Making changes while connected via USB                                                            |
+|      ✅       | Native app for: Windows, Linux, MacOS                                                             |
```

Comment:
Similar question for this one, seems like it shouldn't be in the feature list and is already mentioned below.

Reply:
Yeah, I'm ok if this isn't included in the list. It's not really a "feature" per se.

Reply:
I was thinking of the possible features that Studio could be extended with in the future when mentioning this and the previous point. For completeness I felt it's nice to have them listed in that case.